### PR TITLE
Tgo/feat/navigation

### PR DESCRIPTION
**Pull request by Shiiboun**
Please add branch to development

### CODE REVIEW

**Comment by OlafVanSchlacht** [comment]
Please fix open bugs:
-Border bewteen content and navigation div is not closed
-Give the links a background color for better indication
-The Textfield in the contact form should have a fixed size
-The message label in the contact form should be at the top of the textfield

**Comment by Shiiboun** [comment]
ok

**Comment by Shiiboun** [comment]
Fixed open bugs. Pls review

**Comment by OlafVanSchlacht** [comment]
-Decreasing the width of the textfields is not a valid bug fix. Please take a look at it and find another solution.
-The border under the navigation items is confusing at first sight and looks like it was not intended.
-The placeholder in the registration and contact form is not acceptable.

**Comment by S3bb** [comment]
Toller Pull

File: app.js [hunk 1]
```diff
@@ -21,5 +21,8 @@ app.use('/registration', registration);
 app.get('/', function(req, res) {
   res.render('index');
 });
+app.get('/contact', function(req, res) {
+  res.render('contact')
```
Comment (S3bb) [inline app.js]: Missing semicolon

File: app.js [hunk 2]
```diff
@@ -21,5 +21,8 @@ app.use('/registration', registration);
 app.get('/', function(req, res) {
   res.render('index');
 });
+app.get('/contact', function(req, res) {
```
Comment (S3bb) [inline app.js]: No blank line between blocks?

**Comment by OlafVanSchlacht** [comment]
No more content has been added yet

**Pull request closed without merging**